src/components/NavBar.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { getAllUsers, logoutUser } from '../redux/users';
+import Friends from './Friends'
+import Register from './Register';
+import Login from './Login';
+
+
+
+class NavBar extends Component {
+	constructor() {
+		super()
+		this.handleLogout = this.handleLogout.bind(this)
+	  }
+	async handleLogout(){
+		await this.props.logoutUser()
+	  }
+
+	render() {
+	const {user} = this.props
+		return (
+			<div>
+			<nav id='navBar'>
```

Comment:
be consistent. in another file i saw "x-y" but here it's camelcase. keep it either consistently camelcased or 'x-y'ed